app/models/skill.rb

```diff
@@ -18,4 +18,12 @@ def self.parse_skills
 
   scope :with_skill_type, ->(skill_name) { where(type_of_skill: skill_name) }
 
+  def career_skill
+    (rank.to_i + 1) * 5
```

Comment:
If it's 0 it returns nil, we don't know why though, if you can enlighten us we'll be delighted please thank you.